src/consensus/pbft/pbftmetadata.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I am not sure whether metadata is the best name for this file here. Is the plan to have a file per PBFT table? If yes, I believe that this file should be called `pbftrequests.h`? Otherwise, the tables definitions could be moved to `pbfttables.h`.